Src/BootCamp.Chapter1/Hangman.cs

```diff
@@ -23,18 +24,76 @@ public static class Hangman
         /// <returns>True if game was won or false if game was lost.</returns>
         public static bool Play(int lives, string wordsFile, int difficulty)
         {
-            var wordToGuess = WordsBank.PickRandomWord(@$"Words/{wordsFile}", difficulty);
-            // ToDo: finish the game!
-            var playersGuess = GuessCharacter();
+            if(lives < 1 || difficulty < 3)
+            {
+                throw new SystemException();
```

Comment:
Throw your own exception, always. This is bad for 2 reasons:
1) there is nothing wrong with system. Basic exception class is better suited.
2) why did it fail? You should give all the details needed (lives and difficulty) to explain what went wrong.